.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: Release to dockerhub
+on:
+  push:
+    tags:
+      - dockerhub_release_*  # Launch dockerhub push on dockerhub_release_* tags
+jobs:
+  build_docker_image:
+    runs-on: ubuntu-latest
+    env:
+      registry: osrf/subt-virtual-testbed
+    strategy:
+      matrix:
+        image_name: ['cloudsim_sim', 'cloudsim_bridge', 'subt_shell', 'subt_sim_entry']
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install base dependencies
+        run: |
+          sudo apt update;
+          sudo apt-get install -y docker
```

Comment:
I recommend to check that this line really installs `docker`. This used to work for me on VRX on `ubuntu-latest` but that's no longer the case (maybe something changed in the Ubuntu package?). I installed it using this action:

```
- name: Install Docker
        uses: docker-practice/actions-setup-docker@master
        with:
          docker_version: 18.09
          docker_channel: stable
```